v1.1-beta/matrixx-digital-commerce/matrixx-dcp/cnf-testsuite.yml

```diff
@@ -0,0 +1,111 @@
+---
+helm_directory: "matrixx"
+release_name: "cnf"
+helm_install_namespace: "matrixx"
+helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values1685358688.yaml\
+  \ --devel"
+allowlist_helm_chart_container_names:
+- "falco"
+- "promtail"
+- "prometheus"
+container_names:
+- name: "engine-controller"
+  rollback_from_tag: "5261"
+  rolling_update_test_tag: "5270-SNAPSHOT"
```

Comment:
@deerowbear Is this a product release an end-user can test again?

Reply:
5270 is the current version we are working on. 5261 is a version that is availabe that someone could be on.